Sources/Rendering/OpenGL/glsl/vtkVolumeFS.glsl

```diff
@@ -815,10 +815,22 @@ void computeIndexSpaceValues(out vec3 pos, out vec3 endPos, out float sampleDist
   sampleDistanceIS = sampleDistance*delta2/delta;
 }
 
+uniform int cameraParallel;
+uniform vec3 cameraDirection;
+
 void main()
 {
-  // camera is at 0,0,0 so rayDir for perspective is just the vc coord
-  vec3 rayDirVC = normalize(vertexVCVSOutput);
+
+  vec3 rayDirVC;
+
+  if (cameraParallel == 1)
+  {
+    // Camera is parallel, so the rayDir is just the direction of the camera.
+    rayDirVC = cameraDirection;
+  } else {
```

Comment:
I believe in parallel projection the rayDirVC is always vec3(0.0, 0.0, 1.0) can  you try that?

Reply:
Oh interesting. Its actually always (0,0,-1) in view space. Which is what I was actually calculating haha. I'll update it.

Reply:
Addressed.